src/sha256.ts

```diff
@@ -0,0 +1,5 @@
+import { sha256 as rawSha256 } from "@noble/hashes/sha256"
```

Comment:
Each module should start with a documentation block (file header) that documents the purpose of the module.